backend/src/seeds/graphql/mutations.ts

```diff
@@ -81,9 +81,14 @@ export const createTransactionLink = gql`
 
 // from admin interface
 
-export const createPendingCreation = gql`
+export const adminCreateContribution = gql`
```

Comment:
please do not mix Rights/Rules with method names, especially when with replacement of the elopage process the user will capture his contributions himself and the admin will only confirm them